.dialyzer.ignore-warnings

```diff
@@ -42,8 +42,9 @@
 apps/evm/lib/evm/functions.ex:163: Function 'not_enough_gas?'/2 has no local return
 apps/evm/lib/evm/functions.ex:175: Function 'is_invalid_jump_destination?'/3 will never be called
 apps/evm/lib/evm/functions.ex:185: Function 'static_state_modification?'/2 will never be called
-apps/evm/lib/evm/gas.ex:133: Function cost/2 has no local return
 apps/evm/lib/evm/gas.ex:133: Function cost/3 has no local return
+apps/evm/lib/evm/gas.ex:133: Function cost/2 has no local return
+apps/evm/lib/evm/gas.ex:138: The call 'Elixir.EVM.Gas':operation_cost(atom(),_inputs@1::[integer()],_machine_state@1::#{'__struct__':='Elixir.EVM.MachineState', 'active_words':=integer(), 'gas':=integer(), 'last_return_data':=[integer()], 'memory':=binary(), 'previously_active_words':=integer(), 'program_counter':=integer(), 'stack':=[integer()]},_exec_env@1::#{'__struct__':='Elixir.EVM.ExecEnv', 'account_interface':=atom(), 'address':=<<_:160>>, 'block_interface':=atom(), 'config':=atom(), 'created_accounts':=[<<_:160>>], 'data':=binary(), 'gas_price':=integer(), 'machine_code':=binary(), 'originator':=<<_:160>>, 'sender':=<<_:160>>, 'stack_depth':=integer(), 'static':=boolean(), 'value_in_wei':=integer()}) breaks the contract (atom(),['Elixir.EVM':val()],['Elixir.EVM':val()],'Elixir.EVM.MachineState':t()) -> t() | 'nil'
```

Comment:
Is this a new warning? If so, we avoid introducing new warnings.

Reply:
@germsvel no, it's not new. fields of ExecEnv struct changed